src/main/g8/src/test/scala/$name__camel$/JSDomSpec.scala

```diff
@@ -0,0 +1,23 @@
+package $name;format="camel"$
+
+import org.scalatest._
+import org.scalajs.dom._
+import monix.execution.ExecutionModel.SynchronousExecution
+import monix.execution.schedulers.TrampolineScheduler
+import monix.execution.{Cancelable, Scheduler}
+
+abstract class JSDomSpec extends FlatSpec with Matchers with BeforeAndAfterEach {
```

Comment:
Do you really want to force `FlatSpec` and `Matchers` on your users?

Reply:
Are there any disadvantages with this approach? Feel free to simplify/improve it with a PR.